playbooks/generate_template.yaml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Generate portal template files
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    mcp_roles:
+      - github_mcp
+
+  tasks:
+    - name: Load MCP roles to gain access to defaults
+      ansible.builtin.include_role:
+        name: "{{ item }}"
+        tasks_from: noop # don't know why this doesn't do anything
+        public: true
+        apply:
+          ignore_errors: true # there should be a better way of doing this
```

Comment:
Is this needed or could include_role be sufficient - the way we're doing it in install_mcp.yml?

Reply:
(not vital to figure out right now since it's just maintaining the templates)

Reply:
The issue is that github_mcp has a meta dependency on the common role, which won't run because it's not set up correctly. This is a giant hack to say everything is broken don't worry about it. Ideally the noop tasks file would just noop instead of running common as well